pandas/tests/io/parser/dtypes/test_dtypes_basic.py

```diff
@@ -335,3 +336,29 @@ def test_nullable_int_dtype(all_parsers, any_int_ea_dtype):
     )
     actual = parser.read_csv(StringIO(data), dtype=dtype)
     tm.assert_frame_equal(actual, expected)
+
+
+@pytest.mark.parametrize("default", ["float", "float64"])
+def test_dtypes_defaultdict(all_parsers, default):
```

Comment:
`default` doesn't look used in the test?

Reply:
Thx, missed that